test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/OptionTagHelperTest.cs

```diff
@@ -15,11 +14,11 @@ public class OptionTagHelperTest
     {
         // Original content, selected attribute, value attribute, selected values (to place in TagHelperContext.Items)
         // and expected tag helper output.
-        public static TheoryData<string, string, string, CurrentValues, TagHelperOutput> GeneratesExpectedDataSet
```

Comment:
`CurrentValues` is no longer public, so it cannot be part of a public API surface